mods/cnc/rules/structures.yaml

```diff
@@ -757,15 +757,15 @@ OBLI:
 		Range: 8c0
 	Bib:
 		HasMinibib: Yes
-	WithChargeAnimation:
+	WithAttackAnimation:
+		AttackSequence: active
+	PreparingAttackSounds:
+		Sounds: obelpowr.aud
 	Armament:
 		Weapon: Laser
 		LocalOffset: 0,-85,1280
-		FireDelay: 0
-	AttackCharge:
-		ChargeAudio: obelpowr.aud
-		ReloadDelay: 40
-		InitialChargeDelay: 50
+		FireDelay: 40
```

Comment:
> so this is a pretty bad regression.

I agree, but I'd like to point out that the underlying problem is (or was, as I have a fix for it) that `Armament` schedules the delayed action almost unconditionally, a larger `FireDelay` for other weapons would have the same issue, ignoring the minimum range as well.